spec/features/admin_deletes_edits_sites_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+
+feature "Admin deletes and edits sites", %{
+  As admin I want to edit or delete a site
+  } do
+
+    # Acceptance Criteria:
+    # - When I delete the site, it no longer appears in the site's view
+    #   an error message
+    # - When I edit the site's details, the updated details are shown on the site's page
```

Comment:
Line is too long. [88/80]<br>Incorrect indentation detected (column 4 instead of 3).